### PR TITLE
Adds functionality to aggregate keys by labels

### PR DESCRIPTION
**Pull request by matilote**
- allows to add labels to a set of pubkeys
- creates new grafana dashboard to visualise labeled keys
- adds volume mapping for `config.yaml`

### CODE REVIEW

**Comment by menecio** [comment]
Thanks for the contribution, I will review and test this.

File: grafana/dashboard.json [hunk 1]
```diff
@@ -514,32 +1388,211 @@
...
-    "list": []
+    "list": [
+      {
+        "current": {
+          "selected": true,
+          "text": [
+            "foundation-1-nethermind-prysm"
+          ],
+          "value": [
+            "foundation-1-nethermind-prysm"
```
Comment (menecio) [inline grafana/dashboard.json]: could  you give a generic name to these?

File: grafana/dashboard.json [hunk 2]
```diff
@@ -514,32 +1388,211 @@
...
+          "values": false
+        },
+        "textMode": "value"
+      },
+      "pluginVersion": "10.0.0-cloud.3.b04cc88b",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
```
Comment (menecio) [inline grafana/dashboard.json]: All prometheus datasources seem to be set to your datasource uid, it should be the default.

image

**Review by menecio — CHANGES_REQUESTED** [review]
Thanks for the contribution, I left some small change requests that should be addressed before we can merge this.

File: grafana/dashboard.json (hunk 1 above)
Comment (matilote) [reply grafana/dashboard.json]: Removed those

File: grafana/dashboard.json (hunk 2 above)
Comment (matilote) [reply grafana/dashboard.json]: Fixed with properly externally exported dashboard.json

**Review by menecio — APPROVED** [review]

**Pull request merged**